application/sitebuilder/build_service.py

```diff
@@ -121,18 +124,24 @@ def _delete_files_not_needed_for_deploy(build_dir):
 def _start_build(app, build, session):
     build_exception = None
     try:
+        print("DEBUG _start_build(): Marking build as started...")
         _mark_build_started(build, session)
         do_it(app, build)
+        print("DEBUG _start_build(): Done it!")
         build.status = "DONE"
         build.succeeded_at = datetime.utcnow()
-        session.add(build)
+        session.add(build)  # We shouldn't need this here as it is in finally clause?
```

Comment:
@samuelhwilliams @frankieroberto 
Am I right thinking we shouldn't need this here as the `finally` one will always be called?  I could delete it now?

Reply:
Yep 👍